.buildkite/beta-builds.yml

```diff
@@ -0,0 +1,36 @@
+# This pipeline is meant to be run via the Buildkite API, and is
+# only used for beta builds
+
+# Nodes with values to reuse in the pipeline.
+common_params:
+  # Common plugin settings to use with the `plugins` key.
+  - &common_plugins
+    - automattic/bash-cache#2.1.0
+
+steps:
+
+  - label: "Lint WordPress"
+    command: ".buildkite/commands/lint.sh wordpress"
+    key: wplint
+    artifact_paths:
+      - "**/build/reports/lint-results*.*"
+
+  # - label: "🔬 Lint Jetpack"
+  #   command: ".buildkite/commands/lint.sh jetpack"
+  #   key: jplint
+  #   artifact_paths:
+  #     - "**/build/reports/lint-results*.*"
```

Comment:
@jkmassel I'm assuming that this step was commented because the linter on Jetpack failed, and that the lint failure was due to p8Qyks-351-p2#comment-1298 / https://github.com/wordpress-mobile/WordPress-Android/commit/35103522f8c410f0d4aa81906e97bfd42377f4e9 .

If this assumption is correct, I just wanted to note that I fixed this particular issue in https://github.com/wordpress-mobile/release-toolkit/pull/390. So as soon as we land a new version of the `release-toolkit` and start using it in `WordPress-Android`, I'm assuming we should be able to re-enable that `jplint` step? 🙂